cime_config/testdefs/testlist_cam.xml

```diff
@@ -53,36 +47,27 @@
     </options>
   </test>
 
-<!-- NorESM-CAM6 combination not supported (yet?)
-
-  <test compset="NF1850" grid="ne30pg3_ne30pg3_mtn14" name="ERP_Ln9" testmods="cam/outfrq13m">
+  <test compset="NF1850oslo" grid="ne30pg3_ne30pg3_mtn14" name="ERP_Ln9" testmods="cam/outfrq9s">
     <machines>
-      <machine name="betzy" compiler="intel" category="prealpha"/>
+      <machine name="betzy" compiler="intel" category="aux_cam_noresm"/>
     </machines>
     <options>
       <option name="wallclock">00:20:00</option>
     </options>
   </test>
 
-  <test compset="NF2000climo" grid="ne30pg3_ne30pg3_mtn14" name="ERP_Ln9" testmods="cam/outfrq1d">
-    <machines>
-      <machine name="betzy" compiler="intel" category="prealpha"/>
-    </machines>
-    <options>
-      <option name="wallclock">00:20:00</option>
-    </options>
-  </test>
+  <!-- NorESM-CAM6 combination not supported (these are NorESM physics with CAM60)
 
-  <test compset="NF2000climo" grid="ne3pg3_ne3pg3_mg37" name="ERP_D_Ln9_P36" testmods="cam/outfrq9s">
+  <test compset="NF2000climoC6" grid="ne30pg3_ne30pg3_mtn14" name="ERP_Ln9" testmods="cam/outfrq1d">
     <machines>
-      <machine name="betzy" compiler="intel" category="aux_cam_noresm"/>
+      <machine name="betzy" compiler="intel" category="prealpha"/>
```

Comment:
We are using `prealpha_noresm`.
```suggestion
      <machine name="betzy" compiler="intel" category="prealpha_noresm"/>
```

Reply:
fixed.

Reply:
This is still not fixed everywhere in this file.